Alignment/HIPAlignmentAlgorithm/python/ALCARECOTkAlCosmicsCTF0TTrackSelection_cff_py.txt

```diff
@@ -4,59 +4,70 @@ process.maxEvents = cms.untracked.PSet(
 
 import RecoTracker.TrackProducer.TrackRefitters_cff
 process.TrackRefitter1 = RecoTracker.TrackProducer.TrackRefitterP5_cfi.TrackRefitterP5.clone()
-process.TrackRefitter1.src = '<SKIM>'
+process.TrackRefitter1.src = 'ALCARECOTkAlCosmicsCTF0T'
 process.TrackRefitter1.TrajectoryInEvent = True
-process.TrackRefitter1.TTRHBuilder = "WithAngleAndTemplate"
 process.TrackRefitter1.NavigationSchool = ''
+process.TrackRefitter1.TTRHBuilder = "WithAngleAndTemplate"
+
+#-- initialize beam spot
+process.load("RecoVertex.BeamSpotProducer.BeamSpot_cfi")
+
 
 # parameters for alignmentHitFilter
 process.TrackerTrackHitFilter.src = 'TrackRefitter1'
+process.TrackerTrackHitFilter.useTrajectories= True
+process.TrackerTrackHitFilter.minimumHits = 8
 process.TrackerTrackHitFilter.commands = cms.vstring("keep PXB","keep PXE","keep TIB","keep TID","keep TOB","keep TEC")
-process.TrackerTrackHitFilter.minimumHits = 6
-#process.TrackerTrackHitFilter.detsToIgnore = cms.vuint32()
 process.TrackerTrackHitFilter.replaceWithInactiveHits = True
 process.TrackerTrackHitFilter.stripAllInvalidHits = False
 process.TrackerTrackHitFilter.rejectBadStoNHits = True
-process.TrackerTrackHitFilter.StoNcommands = cms.vstring("ALL 12.0")
-process.TrackerTrackHitFilter.useTrajectories= True
+#process.TrackerTrackHitFilter.StoNcommands = cms.vstring("ALL 12.0")  #deco mode
+process.TrackerTrackHitFilter.StoNcommands = cms.vstring("ALL 18.0")   #peak mode
 process.TrackerTrackHitFilter.rejectLowAngleHits= True
-process.TrackerTrackHitFilter.TrackAngleCut= 0.17 #~10 degrees
+process.TrackerTrackHitFilter.TrackAngleCut= 0.35 # in rads, starting from the module surface; .35 for cosmcics ok, .17 for collision tracks
 process.TrackerTrackHitFilter.usePixelQualityFlag= True
```

Comment:
@CandiceYou In the validation tools we take 0.1 for this cut when using cosmics, if I am not mistaken.
In general, you might consider (in the next update) to use the [unified track selection and refitting sequence](https://github.com/cms-sw/cmssw/blob/CMSSW_8_1_X/Alignment/CommonAlignment/python/tools/trackselectionRefitting.py).